app/views/two_factor_authentication/backup_code_verification/show.html.erb

```diff
@@ -1,4 +1,4 @@
-<% self.title = t('titles.enter_2fa_code.security_code') %>
+<% self.title = t('titles.enter_2fa_code.backup_code') %>
```

Comment:
Maybe we can just reuse the same string as the heading and avoid duplicating it?

```suggestion
<% self.title = t('two_factor_authentication.backup_code_header_text') %>
```

Reply:
True. I was thinking it had to be set from a titles: etc: etc: key but this way is is less of an overall code change.

Reply:
Yeah I could see an argument that it's nice to have all our titles in `titles:`, but I think this scenario has highlighted that it has a tendency to fall out of sync, and I think generally there shouldn't be many situations where the title is different from the main page H1.